BEP92.md

```diff
@@ -0,0 +1,51 @@
+# BEP-92: Implementation of SHA3-256 FIPS 202 hash and EcRecover Uncompressed public key precompiled contracts
+
+- [BEP-92: Implementation of SHA3-256 FIPS 202 hash precompiled and EcRecover Uncompressed public key precompiled](#bep-92)
+  - [1. Summary](#1-summary)
+  - [2. Abstract](#2-abstract)
+  - [3. Status](#3-status)
+  - [4. Motivation](#4-motivation)
+  - [5. Specification](#5-specification)
+  - [6. License](#6-license)
+  
+## 1. Summary
+
+This BEP describes a proposal for Implementation of SHA3-256 as precompiled for BSC
+
+## 2. Abstract
+
+BEP-92 Proposal describes a addition of implementation of SHA3-256 FIPS 202 hash and EcRecover Uncompressed public key methods as precompiled native contracts
+
+## 3. Status
+
+This BEP is a Work in Progress(WIP). 
+
+## 4. Motivation
+
+To bring more interoperability to Binance Smart Chain, this BEP proposing to add additional capabilities to verify sha3-256 based blockchains in the likes of ICON republic.
+
+- In order to validate the merkle tree data from ICON and other similar blockchains, which uses the SHA3-256 hashing function. 
+- When the transaction hashes are created using the SHA3-256 hashing function and the same hashing mechanism is needed to validate signatures of validators in ICON. 
+- Currently ICON uses a similar address building scheme to ethereum. ICON uses a part of the hash value of the public key, but with with SHA3-256 FIPS hashing function. Hence the need for new ecrecoverPublicKey function as part of precompiled.
+
+## 5. Specification
+Ethereum based virtual machines uses Keccak-256 hashing algorithm to evaulate blocks, but most of the modern blockchains adapted SHA3-256,(slightly different padding rule) as part of [NIST FIPS 202](#https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.202.pdf).
+

```

Comment:
Shall we introduce the gas consumption  of these two precompile contracts?

Reply:
@guagualvcha , Ill add the gas details on this.

Reply:
Added, Please let me know if any changes needed on this, cc/ @que1414

Reply:
Thanks.